.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
I suspect this wants to be `pull_request_target` (see https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target). That way forks can't change ci.yml and run things without us auditing it.

Reply:
`pull_request_target` seems to be better, thanks!